Dependencies/xr/Source/OpenXR/XR.cpp

```diff
@@ -813,6 +828,29 @@ namespace xr
                 }
             }
 
+            // Create the eye tracking get pose action, suggested binding, and space
+            if (SupportsEyeTracking)
+            {
+                XrActionCreateInfo actionInfo{ XR_TYPE_ACTION_CREATE_INFO };
+                actionInfo.actionType = XR_ACTION_TYPE_POSE_INPUT;
+                strcpy_s(actionInfo.actionName, ActionResources.EYE_GET_POSE_ACTION_NAME);
+                strcpy_s(actionInfo.localizedActionName, ActionResources.EYE_GET_POSE_ACTION_LOCALIZED_NAME);
+                XrCheck(xrCreateAction(ActionResources.ActionSet, &actionInfo, &ActionResources.EyeGetPoseAction));
+
+                // Create suggested binding
+                std::string path{ "/user/eyes_ext" };
+                path.append(ActionResources.EYE_GET_POSE_PATH_SUFFIX);
```

Comment:
fwiw instead of doing multiple std::string allocations here you could just `sprintf` into a `char actionPath[XR_MAX_PATH_LENGTH]`

Reply:
Yeah, this is kind of ugly, I didn't really clean it up after copying the way controllers were processed